glustercli/cmd/volume-profile.go

```diff
@@ -0,0 +1,125 @@
+			failure(fmt.Sprintf("Failed to get volume profile info for volume %s\n", volname), err, 1)
+		}
+		// Iterate over all bricks
+		for index := range volumeProfileInfo {
+
+			// Display Cumulative Stats
+			table := tablewriter.NewWriter(os.Stdout)
+			fmt.Printf("Brick: %s\n\n", volumeProfileInfo[index].BrickName)
+			if volumeProfileInfo[index].CumulativeStats.Interval != "" {
+				fmt.Printf("Cumulative Stats: \n")
+			}
+			table.SetHeader([]string{"%-Latency", "AvgLatency", "MinLatency", "MaxLatency", "No. Of Calls", "FOP"})
+			if len(volumeProfileInfo[index].CumulativeStats.StatsInfo) != 0 {
+				// Iterate over stats of fop in Cumulative stats, key being the FOP name
+				for key := range volumeProfileInfo[index].CumulativeStats.StatsInfo {
+					table.Append([]string{volumeProfileInfo[index].CumulativeStats.StatsInfo[key]["%-latency"],
+						volumeProfileInfo[index].CumulativeStats.StatsInfo[key]["avglatency"],
+						volumeProfileInfo[index].CumulativeStats.StatsInfo[key]["minlatency"],
+						volumeProfileInfo[index].CumulativeStats.StatsInfo[key]["maxlatency"],
+						volumeProfileInfo[index].CumulativeStats.StatsInfo[key]["hits"],
+						key})
+				}
+				table.Render()
+			}
+			if volumeProfileInfo[index].CumulativeStats.Duration != "" {
+				fmt.Printf("Duration: %s seconds\n", volumeProfileInfo[index].CumulativeStats.Duration)
+				fmt.Printf("Data Read: %s bytes\n", volumeProfileInfo[index].CumulativeStats.DataRead)
+				fmt.Printf("Data Write: %s bytes\n\n\n", volumeProfileInfo[index].CumulativeStats.DataWrite)
+			}
+			fmt.Printf("\n\n")
+
+			// Display Interval Stats
+			table = tablewriter.NewWriter(os.Stdout)
+			table.SetHeader([]string{"%-Latency", "AvgLatency", "MinLatency", "MaxLatency", "No. Of Calls", "FOP"})
+			if volumeProfileInfo[index].IntervalStats.Interval != "" {
+				fmt.Printf("Interval %s Stats: \n\n", volumeProfileInfo[index].IntervalStats.Interval)
+			}
+			if len(volumeProfileInfo[index].IntervalStats.StatsInfo) != 0 {
+				// Iterate over stats of fop in Cumulative stats, key being the FOP name
+				for key := range volumeProfileInfo[index].IntervalStats.StatsInfo {
+					table.Append([]string{volumeProfileInfo[index].IntervalStats.StatsInfo[key]["%-latency"],
+						volumeProfileInfo[index].IntervalStats.StatsInfo[key]["avglatency"],
+						volumeProfileInfo[index].IntervalStats.StatsInfo[key]["minlatency"],
+						volumeProfileInfo[index].IntervalStats.StatsInfo[key]["maxlatency"],
+						volumeProfileInfo[index].IntervalStats.StatsInfo[key]["hits"],
+						key})
+				}
+				table.Render()
+			}
+			if volumeProfileInfo[index].IntervalStats.Duration != "" {
+				fmt.Printf("Duration: %s seconds\n", volumeProfileInfo[index].IntervalStats.Duration)
+				fmt.Printf("Data Read: %s bytes\n", volumeProfileInfo[index].IntervalStats.DataRead)
+				fmt.Printf("Data Write: %s bytes\n", volumeProfileInfo[index].IntervalStats.DataWrite)
+			}
+			fmt.Printf("\n\n")
```

Comment:
I think one extra line `\n` is enough

Reply:
@Madhu-1  There will be a lot of data at one point to display, that's why just to keep the output clear enough I decided to go with two extra lines.